home.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1, minimum-scale=1">
+  <title>Senior Swift - Concise weekly blog covering skills you need day to day that employers want</title>
+  <link rel="stylesheet" href="styles.css">
+
+  <link href="https://fonts.googleapis.com/css?family=Montserrat|Playball|Playfair+Display:700" rel="stylesheet">
+</head>
+
+<body class="home-page">
```

Comment:
You don't use class home-page in your css. Instead you select a class .body that doesn't exist in your html.
Suggest you use the simple type selector body instead in your css - and remove the home-page class if it's not needed for anything you can't accomplish with the body selector...